config/paths.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var path = require('path');
```

Comment:
Since we're no longer supporting older versions of Node, we should be able to replace all instances of `var` with `const` (and `let` if needed).